README.md

```diff
@@ -444,16 +444,35 @@ Mastering methods are:
     - `forceLength` (int) - force the audio length of the mastered track (in seconds).
     - `audience` (list) - List of dicts containing the personalisation parameters. This parameter depends on the number of parameters you used in your [script](#script) resource. In the script documentation example above, we used 2 parameters: `username` and `location`, and in the following example below we want to produce the script for username `Antonio` with location `Barcelona`. If audience is not provided, the fallback track will be created.
     - `mediaFiles` (list) - List of dicts containing the media files. This parameter depends on the media file tags used in the [script](#script) resource and the media files you have in your account. For example, if the script contains `<<media::myrecording>>` plus `<<media::mysong>>`, and you want to attach myrecording to mediaId = "12345", and mysong to mediaId = "67890" then `mediaFiles = [{"myrecording":"12345", "mysong":"67890"}]`.
+    - `sectionProperties` (dict) - This variable takes a dict as its input, whereby each key has the name of the *section* and its *value* is another dict with the properties corresponding to that section. We currently support two properties per section; endAt: (float) the time in seconds that this section should end at, and justify: (str) [optional] how the audio content of this section should be justified. This feature is explained in depth [here.](https://anvilproject.org/guides/content/creating-links) 
+
+    - `chanels` (int) - Specifies the number of channels in the mastered file. You can either use 1 for mono or 2 for stereo (default)
```

Comment:
fix typo

Reply:
fix typo in microservice as well if it exists

Reply:
fixewd